paper-collapse-item.html

```diff
@@ -136,8 +136,14 @@
 		},
 
 		// Private methods
+		/**
+		 * Fired whever the collpase status is changed
```

Comment:
Could you please fix typos? Or better also align naming to "Fired whenever the open status is changed"?